spec/features/article_searching_spec.rb

```diff
@@ -39,6 +39,28 @@
       click_button 'Search'
     end
 
+    expect(page).to have_css('h2', text: /\d+ results?/)
+
     expect(current_url).to match(%r{/article\?.*&q=Kittens})
   end
+
+  scenario 'article records are navigable from search results' do
+    stub_article_service(docs: [SolrDocument.new(id: 'abc123', eds_title: 'The title of the document')])
+    stub_article_service(type: :single, docs: [SolrDocument.new(id: 'abc123', eds_title: 'The title of the document')])
```

Comment:
The first is for the search results and the second is for the document view (hence the `type: :single`) because the search service either received `search_results` or `fetch`.

I'll update the SolrDocuments that I'm using to stub so that is more obvious.